### DISCUSSION TITLE
Updated tutorials for Fabric?

### DISCUSSION THREAD

**Discussion by SirEnder125** [FabricMC/fabric-api#1715] [Mod Dev Support] [answered]
Hey. I was recently trying to create a custom projectile, but as I neared the end of the tutorial, I found out that some of the functions/classes used were deprecated. So is there any place that an updated mod is uploaded? I'm trying to make a grenade in Minecraft 1.17.1.

Here's a link to the outdated tutorial: https://fabricmc.net/wiki/tutorial:projectiles.

**Comment by Daomephsta** [comment, marked as answer]
No documentation related to Minecraft itself is ever fully up to date. If you find an out of date tutorial, report it here or in #wiki on the discord, and someone will get around to updating it.

**Reply by SirEnder125** [reply]
Oh, alright. Understood. Thanks! Is my question enough to maybe get someone started on updating it?

**Reply by Daomephsta** [reply]
It would be helpful if you linked to the out of date tutorial.

**Reply by SirEnder125** [reply]
Alright. Thanks. :D
